_data/sites.json

```diff
@@ -2486,6 +2487,18 @@
         ]
     },
 
+    {
+      "name": "CDEK Forward",
+      "url": "https://global.cdek.ru",
```

Comment:
I suggest that we should use the U.S. URL of CDEK forward as the main URL instead.

```suggestion
      "url": "https://cdekforward.com/us/",
```

Reply:
I don't know, googling CDEK forward the first hit is the RU one, so I will disagree here.